src/main/java/emissary/command/ServiceCommand.java

```diff
@@ -117,13 +125,23 @@ protected void stopService() {
     /**
      * Send a get request using the {@link EmissaryClient}
      *
-     * @param actionEndpoint the endpoint i.e. /api/health
+     * @param endpoint the endpoint i.e. /api/health
+     * @return the response object
+     */
+    protected EmissaryResponse performGet(String endpoint) {
+        return new EmissaryClient().send(new HttpGet(getEndpoint(endpoint)));
+    }
+
+    /**
+     * Send a get request using the {@link EmissaryClient}
+     *
+     * @param endpoint the endpoint i.e. /api/health
      * @return the response object
      */
-    protected EmissaryResponse performAction(String actionEndpoint) {
+    protected EmissaryResponse performPost(String endpoint) {
```

Comment:
performPost, performGet and getEndpoint should probably all move to the HttpCommand

Reply:
good point...moved the methods to HttpCommand